searchlib/src/vespa/searchlib/query/streaming/in_term.cpp

```diff
@@ -21,8 +21,9 @@ InTerm::InTerm(std::unique_ptr<QueryNodeResultBase> result_base, const string &
 InTerm::~InTerm() = default;
 
 void
-InTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data)
+InTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data, const fef::IIndexEnvironment& index_env)
 {
+    (void) index_env;
```

Comment:
When the type is specific you do not need a name for the parameter, and then you do not need the explicit (void).